Documentation/ExtensionArchitecture/Extbase/Reference/Validation/Validators/Index.rst

```diff
@@ -0,0 +1,648 @@
+:navigation-title: List of Validators
+
+..  include:: /Includes.rst.txt
+..  _extbase-validator:
+
+=======================================
+Built-in validators provided by Extbase
+=======================================
+
+This document lists all built-in Extbase validators,
+along with example usage for each using PHP attributes.
+
+..  seealso::
+    *   `Using validation for Extbase models and controllers <https://docs.typo3.org/permalink/t3coreapi:extbase-validation>`_
+    *   `Custom Extbase validator implementation <https://docs.typo3.org/permalink/t3coreapi:extbase-domain-validator>`_
+
+..  note::
+    All validators except the `NotEmptyValidator <https://docs.typo3.org/permalink/t3coreapi:extbase-validator-notempty>`_
+    accept empty values as valid. If empty values should not be possible,
+    combine these validators with the `NotEmptyValidator`.
+
+..  contents:: Validators in Extbase
+
+..  _extbase-validator-alphanumeric:
+
+AlphanumericValidator
```

Comment:
Shouldn't we use `:php-short:` for every validator on this page? Maybe not necessarily in the headline but at least by mentioning it once more by name in the first paragraph as done with various validators here already.

Reply:
Done